apps/qubit/modules/informationobject/templates/storageLocationsCriteria.php

```diff
@@ -5,27 +5,19 @@
 <?php end_slot() ?>
 
 <?php slot('title') ?>
-  <h1><?php echo __('Item list - report criteria') ?></h1>
+  <h1><?php echo __($type.' - report criteria') ?></h1>
```

Comment:
no worries! It'll save time when I come fix it soon if I know where all I did the mistake hehe